docs/channel-messenger-handover-protocol.md

```diff
@@ -11,7 +11,7 @@ Here comes the Messenger handover protocol, which aims to make a smooth transiti
 
 > **Note:**
 >
-> - If you are curious how to change the profile image between Messenger Bot and human agents, please check Bottender's doc, [Using Persona](https://bottender.js.org/docs/channel-messenger-persona).
+> - If you are curious how to change the profile image between Messenger Bot and human agents, please check Bottender's doc, [Using Persona](channel-messenger-persona).
```

Comment:
https://docusaurus.io/docs/en/doc-markdown#linking-other-documents

Docusaurus recommend using this way: `[This links to another document](other-document.md)`.